packages/shared/src/hooks/useExtensionPermission.ts

```diff
@@ -1,49 +0,0 @@
-import { QueryClient, useQueryClient } from '@tanstack/react-query';
```

Comment:
Is this file deleted already? Looks like the content was just removed but not the file entirely.

Reply:
great catch, not sure how this happened, maybe some merge conflict